image_generation/stable_diffusion_1_5/cpp/scripts/set_up_and_run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e  # Exit immediately if a command exits with a non-zero status
+
+abs_path() {
+    script_path=$(eval echo "${BASH_SOURCE[0]}")
+    directory=$(dirname "$script_path")
+    builtin cd "$directory" || exit
+    pwd -P
+}
+cd "`abs_path`"
+
+mkdir ./ov/
+curl https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.1/linux/l_openvino_toolkit_ubuntu20_2023.1.0.12185.47b736f63ed_x86_64.tgz | tar --directory ./ov/ --strip-components 1 -xz
```

Comment:
can we use newer OpenVINO version?

Reply:
will update this with latest release link